sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -1,205 +1,248 @@
 b                   	int                 	                    
 c                   	string              	                    
-c                   	string              	                    
 d                   	string              	                    
+# Partition Information	                    	                    
+# col_name          	data_type           	comment             
+c                   	string              	                    
 d                   	string
 
 
 -- !query 7
-DESC t PARTITION (c='Us', d=1)
+DESC FORMATTED t
 -- !query 7 schema
 struct<col_name:string,data_type:string,comment:string>
 -- !query 7 output
-# Partition Information	                    	                    
-# col_name          	data_type           	comment             
 a                   	string              	                    
 b                   	int                 	                    
 c                   	string              	                    
+d                   	string              	                    
+# Partition Information	                    	                    
+# col_name          	data_type           	comment             
 c                   	string              	                    
 d                   	string              	                    
-d                   	string
+                    	                    	                    
+# Detailed Table Information	                    	                    
+Table               	`default`.`t`       	                    
+Created[removed by SQLQueryTestSuite]
+Last Access[removed by SQLQueryTestSuite]
+Type                	MANAGED             	                    
+Provider            	parquet             	                    
+Num Buckets         	2                   	                    
+Bucket Columns      	[`a`]               	                    
+Sort Columns        	[`b`]               	                    
+Comment             	table_comment       	                    
+Location[removed by SQLQueryTestSuite]sql/core/spark-warehouse/t	                    
+Partition Provider  	Catalog             	                    
+Partition Columns   	[`c`, `d`]          	                    
+Schema              	root
+ |-- a: string (nullable = true)
+ |-- b: integer (nullable = true)
+ |-- c: string (nullable = true)
+ |-- d: string (nullable = true)
 
 
 -- !query 8
-DESC EXTENDED t PARTITION (c='Us', d=1)
+DESC EXTENDED t
 -- !query 8 schema
 struct<col_name:string,data_type:string,comment:string>
 -- !query 8 output
-# Partition Information	                    	                    
-# col_name          	data_type           	comment             
-Detailed Partition Information CatalogPartition(
-	Partition Values: [c=Us, d=1]
-	Storage(Location: sql/core/spark-warehouse/t/c=Us/d=1)
-	Partition Parameters:{})	                    	                    
 a                   	string              	                    
```

Comment:
we should add a header here `# col_name          	data_type           	comment`.